### PR TITLE
Create Feature/base abstract css

### PR DESCRIPTION
**Pull request by Vinh2410**
Add reset.css, typography.css, variable.css, main.css

### CODE REVIEW

File: practice-one/src/assets/css/abstracts/variable.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    --lh-huge: 48px;
+    --lh-large: 27px;
+    --lh-medium: 26px;
+    --lh-small: 24px;
+
+    --fw-primary: 400px;
+    --fw-secondary: 700px;
+
+    --primary-color: #EBCB80;
```
Comment (dieule-agilityio) [inline practice-one/src/assets/css/abstracts/variable.css]: Should lower case the color code to make it consistent

File: practice-one/src/assets/css/abstracts/variable.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    --seconday-color: #4b4b4b;
```
Comment (dieule-agilityio) [inline practice-one/src/assets/css/abstracts/variable.css]: typo

File: practice-one/src/assets/css/abstracts/variable.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    --grey-color: #999999;
```
Comment (dieule-agilityio) [inline practice-one/src/assets/css/abstracts/variable.css:23]: your are naming the color by primary, secondary but why is it grey, darkwhite,... now

File: practice-one/src/assets/css/base/typography.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
+@font-face {
```
Comment (dieule-agilityio) [inline practice-one/src/assets/css/base/typography.css:1]: It's not typography

Comment (Vinh2410) [inline practice-one/src/assets/css/base/typography.css:1]: Is it global.css ?

**Pull request merged**